internal/boxcli/generate.go

```diff
@@ -139,13 +144,17 @@ func runGenerateCmd(cmd *cobra.Command, flags *generateCmdFlags) error {
 	if err != nil {
 		return errors.WithStack(err)
 	}
+	generateOpts := devopt.GenerateOpts{
```

Comment:
Hack, if you embed `devopt.GenerateOpts` to the flag struct you can pass `flags.GenerateOpts` into the functions below.